rego/Commoncontrols.rego

```diff
@@ -713,6 +713,7 @@ tests contains {
 #--
 tests contains {
     "PolicyId": "GWS.COMMONCONTROLS.7.1v0.1",
+    "Prerequisites": ["directory/v1/users/list"],
```

Comment:
Potential headaches acknowledged. I considered using `Commandlet` over here as well, but I didn't like it because we're technically not using `Commandlet`s for ScubaGoggles. Frankly I think `Prerequisites` would be a better fit for ScubaGear as well, not that that's going to change any time soon. So my vote would be to stick with `Prerequisites` on ScubaGoggles, but if the other reviewers have opinions on this and I'm outvoted that's ok too.

Reply:
Just curious in terms of "API speak" would this be considered the "URL" component of the API call? Might not help since we don't (yet) have full coverage via the policy API for all in-scope settings.

Reply:
In this case, yes. The reporter takes this URL and inserts a link to directly to Googles's documentation for it. In this case the full URL displayed in the report would be [directory/v1/users/list](https://developers.google.com/admin-sdk/directory/reference/rest/v1/users/list).

It's worth noting that though that the prerequisites aren't always API calls. For example, the SPF, DKIM, and DMARC controls also depend on custom functions like `get_spf_records`.